android/app/src/main/res/layout/forecast_timestep.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/forecastTimeStep"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="6dp"
+    android:layout_marginBottom="6dp"
+    android:orientation="horizontal">
+
+    <LinearLayout android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+        <TextView
+            android:id="@+id/timeTextView"
+            style="@style/time"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal|top"
+            tools:text="13:00" />
+
+        <ImageView
+            android:id="@+id/weatherIconImageView"
+            android:layout_width="36dp"
+            android:layout_height="36dp"
+            android:layout_gravity="center_horizontal"
+            android:contentDescription=""
+            tools:src="@drawable/s_32_light" />
+
+        <TextView
+            android:id="@+id/temperatureTextView"
+            style="@style/temperatureSmall"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            tools:text="+12°" />
+    </LinearLayout>
+    <!-- Note: 'View' cannot be used with widgets, therefore ImageView here -->
+    <ImageView
+        android:id="@+id/forecastBorder"
+        style="@style/verticalLine"
+        android:layout_width="1dp"
+        android:layout_height="match_parent" />
```

Comment:
Vertical line to short now.
![Screenshot 2024-12-12 at 8 42 11](https://github.com/user-attachments/assets/d1a5f5aa-b646-4ca2-bd02-03b2fd5eac91)